tests/testplan.cpp

```diff
@@ -458,7 +456,7 @@ void TestPlan::testVpmbMetric45m30minTx()
 	while(!dp->minimum_gas.mbar && dp->next) dp = dp->next;
 	QCOMPARE(lrint(dp->minimum_gas.mbar / 1000.0), 108l);
 	// print first ceiling
-	printf("First ceiling %.1f m\n", (mbar_to_depth(first_ceiling_pressure.mbar, &displayed_dive) * 0.001));
+//	printf("First ceiling %.1f m\n", (mbar_to_depth(deco_state->first_ceiling_pressure.mbar, &displayed_dive) * 0.001));
 	// check benchmark run time of 141 minutes, and known Subsurface runtime of 139 minutes
```

Comment:
I do not understand this change. 1) the first_ceiling_pressure is moved to the struct, so the code is corrected for that but 2) it is commented out afterwards. Comment valid for all commented lines in this file.